### DISCUSSION TITLE
Assisting with development: are Mesermerizer boards available?

### DISCUSSION THREAD

**Discussion by MikeSeiler** [PlummersSoftwareLLC/NightDriverStrip#278] [General]
Hello,
I frequently use the ESP32 in my projects.
I have plenty of C/C++ and Python experience.  React is not my favorite.
Where do I find the thread which explains how to get a Mesermerizer board as a potential contributor here?
- Mike

**Comment by Bun-Bun2** [comment]
First paragraph of the readme has the information you are looking for. 

> DEVELOPERS WANTED! We are searching for talented React and C++ developers to help out on this project. Check out the code and if you're interest, contact [davepl@davepl.com](mailto:davepl@davepl.com).
